src/lib/utils/typeChecking.ts

```diff
@@ -0,0 +1,77 @@
+import { CodapLanguageType } from "../../transformers/types";
+import { evalExpression } from "../codapPhone";
+import { prettyPrintCase } from "./prettyPrint";
+
+/**
+ * Enum representing all type checking predicates in CODAP
+ */
+enum CodapTypePredicateFunctions {
+  Boolean = "isBoolean",
+  String = "!isBoundary",
```

Comment:
This is a bit suspect, but CODAP doesn't provide an `isString` so there's not much better we can do. Do you all have thoughts on other things that shouldn't be considered strings?

Reply:
Hmm, yeah I guess this is tricky because string is sort of the catch-all category for CODAP. It still feels a little weird if we were to bake this in though as opposed to them providing it--maybe we should ask Concord folks about adding this.